caim_base/static/js/common.js

```diff
@@ -46,6 +46,47 @@ $('[data-timestamp]').each(function (idx, el) {
     el.text(fmt.format(dte));
 });
 
+$('.reply').click(function(button){
```

Comment:
I think this should really be done in the domready callback? https://api.jquery.com/ready/

Reply:
I don't follow. Do you mean I should add the event inside a JQuery-ready function?